src/pages/CoinPage.jsx

```diff
@@ -1,13 +1,13 @@
 import React, { useState, useEffect, useContext } from "react";
 import { useHistory } from "react-router-dom";
-import axios from "axios";
 import styled from "styled-components";
 import { useParams } from "react-router-dom";
+import { v4 as uuidv4 } from "uuid";
```

Comment:
Sort the imports as follow, absolute imports (like react, npm packages), components and finally styles.